<ui version="4.0" >
 <class>ResourcesWidget</class>
 <widget class="QWidget" name="ResourcesWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QComboBox" name="viewComboBox" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text" >
         <string>Sources</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Files</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sheetComboBox" />
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>2</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="sourcesTreeWidget" >
       <property name="mouseTracking" >
        <bool>false</bool>
       </property>
       <property name="editTriggers" >
        <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors" >
        <bool>true</bool>
       </property>
       <property name="iconSize" >
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="textElideMode" >
        <enum>Qt::ElideMiddle</enum>
       </property>
       <property name="sortingEnabled" >
        <bool>false</bool>
       </property>
       <property name="animated" >
        <bool>false</bool>
       </property>
       <column>
        <property name="text" >
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Length</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Start</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>End</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
